### DISCUSSION TITLE
rbatis for support iOS

### DISCUSSION THREAD

**Discussion by GloomyMeng** [rbatis/rbatis#276] [General]
when i use rbatis for a iOS app with sqlite. but got some error. 
because rbatis_core dependence 'mac_address' supported platforms: Linux, Windows, MacOS, FreeBSD  
so any help for this issues?

**Comment by GloomyMeng** [comment]
cargo dependencies is:
```
rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
```

**Comment by zhuxiujia** [comment]
> cargo dependencies is:
> 
> ```
> rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
> ```

Yes, this dependency will be removed in the next version

**Comment by GloomyMeng** [comment]
> > cargo dependencies is:
> > ```
> > rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
> > ```
> 
> Yes, this dependency will be removed in the next version

so i had to wait a new version for used?
Is there a clear time point?

**Comment by zhuxiujia** [comment]
> > > cargo dependencies is:
> > > ```
> > > rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
> > > ```
> > 
> > 
> > Yes, this dependency will be removed in the next version
> 
> so i had to wait a new version for used? Is there a clear time point?

You may wait about two days。During this time, you can first attribute the use of other operating systems.

**Comment by GloomyMeng** [comment]
> > > > cargo dependencies is:
> > > > ```
> > > > rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
> > > > ```
> > > 
> > > 
> > > Yes, this dependency will be removed in the next version
> > 
> > 
> > so i had to wait a new version for used? Is there a clear time point?
> 
> You may wait about two days。During this time, you can first attribute the use of other operating systems.

ok ~ tks.

**Comment by zhuxiujia** [comment]
> > > > > cargo dependencies is:
> > > > > ```
> > > > > rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
> > > > > ```
> > > > 
> > > > 
> > > > Yes, this dependency will be removed in the next version
> > > 
> > > 
> > > so i had to wait a new version for used? Is there a clear time point?
> > 
> > 
> > You may wait about two days。During this time, you can first attribute the use of other operating systems.
> 
> ok ~ tks.

you can  run  ```  cargo update ```   with try  rbatis = "3.0"   the last  version for IOS。 Please tell me if it is running success！

**Comment by GloomyMeng** [comment]
> > > > > > cargo dependencies is:
> > > > > > ```
> > > > > > rbatis = { version = "3.0.9", default-features = false, features = ["sqlite", "runtime-tokio-native-tls"] }
> > > > > > ```
> > > > > 
> > > > > 
> > > > > Yes, this dependency will be removed in the next version
> > > > 
> > > > 
> > > > so i had to wait a new version for used? Is there a clear time point?
> > > 
> > > 
> > > You may wait about two days。During this time, you can first attribute the use of other operating systems.
> > 
> > 
> > ok ~ tks.
> 
> you can run ` cargo update` with try rbatis = "3.0" the last version for IOS。 Please tell me if it is running success！

iOS build succeed. tks ~
